lib/cog/commands/helpers.ex

```diff
@@ -133,6 +133,8 @@ defmodule Cog.Commands.Helpers do
     do: "Could not find '#{resource_type}' with the name '#{resource_name}'"
   def error(:invalid_handle),
     do: "Invalid chat handle"
+  def error({:trigger_invalid, %Ecto.Changeset{}=changeset}),
+    do: changeset_errors(changeset)
```

Comment:
This doesn't have to be part of this PR, but it looks like we're including this `:trigger_invalid` symbol in the return value when we could probably just return the `{:error, changeset}`.